### ISSUE TITLE
QuadPlane Configuration - VT_TRANS_THR does not exist

### ISSUE THREAD

**Issue by hamishwillee** [PX4/PX4-user_guide#490]
[QuadPlane Configuration > Transition throttle](https://docs.px4.io/master/en/config_vtol/vtol_quad_configuration.html#transition-throttle)

This links to VT_TRANS_THR which does not exist. Did it get replaced or disappear?

@RomanBapst Is this a question for you?

**Comment by xdwgood** [comment]
@hamishwillee 

`VT_F_TRANS_THR`---> Target throttle value for the transition to fixed wing flight.
`VT_B_TRANS_THR`---> Target throttle value for the transition to hover(Multi-rotor) flight

But I didn't find the setting of transmission throttle in tailsitter type

**Comment by hamishwillee** [comment]
Thanks. We need @RomanBapst to comment - it isn't just a matter of replacing the existing parameter in docs.

**Comment by xdwgood** [comment]
Yes, I already know that in tailsitter drones, the thrust when switching to fixed-wing mode is generated by multi-rotor altitude control
Instead of a parameter, we do need to clarify some instructions :smiley:

**Comment by hamishwillee** [comment]
The current docs at this location include a [working link to the parameter](https://docs.px4.io/main/en/config_vtol/vtol_quad_configuration.html#transition-throttle). So I presume this has been updated.

Closing.
